src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/retriever/arcaddon/arc_add_on_retriever.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from ...entity.CustomerResource import CustomerResource
+from ...entity._credentials import Credentials
+from ...exception import DataNotFoundException
+from ...executor.azcli._AzCliExecutor import AzCliExecutor
+from ...constants import Constant
+from ...entity.AzCli import AzCli
+from ...retriever._retriever import Retriever
+
+
+class ArcAddOnRetriever(Retriever):
+    instance = None
+    _arc_add_on_url = Constant.MGMT_URL + "addons" + "?" + Constant.API_VERSION + "=" + Constant.PREVIEW_API_VERSION_VALUE
```

Comment:
This is addons url now, not arc addons url

Reply:
changed to addon